ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/linear_optimisation/algorithms/fillers/RaUsageLimitsFiller.java

```diff
@@ -78,15 +104,23 @@ private void buildIsVariationVariableAndConstraints(LinearProblem linearProblem,
             throw new FaraoException(format("Range action setpoint variable for %s has not been defined yet.", rangeAction.getId()));
         }
 
+        double initialSetpointRelaxation = getInitialSetpointRelaxation(rangeAction);
+
         // range action setpoint <= intial setpoint + isVariationVariable * (max setpoint - initial setpoint)
-        MPConstraint constraintUp = linearProblem.addIsVariationInDirectionConstraint(-LinearProblem.infinity(), initialSetpoint, rangeAction, LinearProblem.VariationReferenceExtension.PREPERIMETER, LinearProblem.VariationDirectionExtension.UPWARD);
+        // RANGE_ACTION_SETPOINT_EPSILON is used to mitigate rounding issues, ensuring the initial situation is feasible
+        // initialSetpointRelaxation is also used for the same purpose in some cases
+        double relaxedInitialSetpoint = initialSetpoint + initialSetpointRelaxation;
+        MPConstraint constraintUp = linearProblem.addIsVariationInDirectionConstraint(-LinearProblem.infinity(), relaxedInitialSetpoint, rangeAction, LinearProblem.VariationReferenceExtension.PREPERIMETER, LinearProblem.VariationDirectionExtension.UPWARD);
         constraintUp.setCoefficient(setpointVariable, 1);
-        constraintUp.setCoefficient(isVariationVariable, -(rangeAction.getMaxAdmissibleSetpoint(initialSetpoint) - initialSetpoint));
+        constraintUp.setCoefficient(isVariationVariable, -(rangeAction.getMaxAdmissibleSetpoint(initialSetpoint) + RANGE_ACTION_SETPOINT_EPSILON - relaxedInitialSetpoint));
 
         // range action setpoint >= intial setpoint - isVariationVariable * (initial setpoint - min setpoint)
-        MPConstraint constraintDown = linearProblem.addIsVariationInDirectionConstraint(initialSetpoint, LinearProblem.infinity(), rangeAction, LinearProblem.VariationReferenceExtension.PREPERIMETER, LinearProblem.VariationDirectionExtension.DOWNWARD);
+        // RANGE_ACTION_SETPOINT_EPSILON is used to mitigate rounding issues, ensuring the initial situation is feasible
+        // initialSetpointRelaxation is also used for the same purpose in some cases
+        relaxedInitialSetpoint = initialSetpoint - initialSetpointRelaxation;
+        MPConstraint constraintDown = linearProblem.addIsVariationInDirectionConstraint(relaxedInitialSetpoint, LinearProblem.infinity(), rangeAction, LinearProblem.VariationReferenceExtension.PREPERIMETER, LinearProblem.VariationDirectionExtension.DOWNWARD);
         constraintDown.setCoefficient(setpointVariable, 1);
-        constraintDown.setCoefficient(isVariationVariable, initialSetpoint - rangeAction.getMinAdmissibleSetpoint(initialSetpoint));
+        constraintDown.setCoefficient(isVariationVariable, relaxedInitialSetpoint - rangeAction.getMinAdmissibleSetpoint(initialSetpoint) - RANGE_ACTION_SETPOINT_EPSILON);
```

Comment:
It think there is a sign error on the epsilon, it should be :
`         constraintDown.setCoefficient(isVariationVariable, relaxedInitialSetpoint - rangeAction.getMinAdmissibleSetpoint(initialSetpoint) + RANGE_ACTION_SETPOINT_EPSILON);`

or 
`
         constraintUp.setCoefficient(isVariationVariable, -(rangeAction.getMinAdmissibleSetpoint(initialSetpoint) - RANGE_ACTION_SETPOINT_EPSILON - relaxedInitialSetpoint));`